_posts/2021/01/2021-01-28-release-notes.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+date: 2021-01-28
+title: "0.15: Release Notes"
+author: olenagerasimova
+description: |
+  Key features and improvements description in the recent
+  version 0.15, released just a few hours ago.
+keywords:
+  - artipie
+  - binary artifacts
+  - hosting of artifacts
+  - docker-adapter
+  - helm-adapter
+  - debian-adapter
+---
+
+[v0.NN](https://example/link) was released on January NN. 
+We are happy to announce that minimal support for [Debian repository](https://wiki.debian.org/DebianRepository) was added to Artipie:
+you can [configure](https://github.com/artipie/artipie/tree/master/examples/debian) the repository 
+which [`apt-get`](https://en.wikipedia.org/wiki/APT_(software)) will be able to work with. 
+
+[Npm-adapter](https://github.com/artipie/npm-adapter) functionality was extended with the following commands support:
+- [`npm dist-tag`](https://github.com/artipie/npm-adapter/issues/155) for managing tags of package versions
+- [`npm deprecate`](https://github.com/artipie/npm-adapter/issues/156) for deprecating and un-deprecating the package
+- [`npm unpublish`](https://github.com/artipie/npm-adapter/issues/154) to un-publish entire package
```

Comment:
@olenagerasimova thanks, one concern. Is it necessary hyphen in `un-deprecating` and `un-publish` words?

Reply:
@genryxy I do not think so as grammar checker says these worlds (not npm commands) should be written with hyphens.